.evergreen/generate_evergreen_tasks.js

```diff
@@ -114,6 +114,22 @@ BASE_TASKS.push({
   ]
 });
 
+BASE_TASKS.push({
+  name: `test-x509-authentication`,
```

Comment:
```suggestion
  name: `test-latest-x509-authentication`,
```

Should this include latest in the name?

Reply:
No, we only test on latest.